var/spack/repos/builtin/packages/clfft/package.py

```diff
@@ -20,6 +20,10 @@ class Clfft(CMakePackage):
     depends_on('opencl@1.2:')
     depends_on('boost@1.33.0:', when='+client')
 
+    # This patch is required until the following PRs are merged.
+    # https://github.com/clMathLibraries/clFFT/pull/229
```

Comment:
Should we just use this patch instead?

Reply:
Is that a patch I created?

Reply:
What I'm saying is that the patch referenced in this PR is more robust than the simple patch you added. What we could do is use:
```python
patch('https://github.com/clMathLibraries/clFFT/commit/eea7dbc888367b8dbea602ba539eb1a9cbc118d9.patch',
      sha256='3148d5937077def301b30b913bc2437df869204fca1de4385ccd46e3b98b13aa', when='@2.12.2')
```

Reply:
I understand.
I fix it to use the above patch.